railties/test/generators/app_generator_test.rb

```diff
@@ -875,8 +875,8 @@ def test_edge_option_during_alpha
     assert_file "Gemfile", %r{^gem\s+["']rails["'],\s+github:\s+["']#{Regexp.escape("rails/rails")}["'],\s+branch:\s+["']main["']$}
   end
 
-  def test_master_option
```

Comment:
Previously the test name was the same as the test below (https://github.com/rails/rails/pull/42270/files#diff-9dd1c2e31fd5617506e344d6328e0e165614290478e1180ef0630017b9305ba4L883), which does not seem right. I have updated it to state we are using the alias.

This test uses an alias and the test below we directly pass in the kwarg to the generator method.